client/app/components/partials/ExportVariants.vue

```diff
@@ -4,36 +4,36 @@
 
 
 .variant-file-body
-  padding-top: 0px
+  padding-top: 0
   padding-bottom: 18px
   margin-bottom: 20px
   margin-top: 20px
 
   .v-input--radio-group__input
     label
-      margin-bottom: 0px
+      margin-bottom: 0
 
   .radio-group.radio-group--column
-    margin-top: 0px
-    padding-top: 0px
-    padding-bottom: 0px
+    margin-top: 0
+    padding-top: 0
+    padding-bottom: 0
```

Comment:
This is not related to the issue or is causing any side effects but I was just wondering if these changes are caused by IDE/ linting?

Reply:
These were done by hand, I've been going through and trying to clean up some of the ide warning messages.  I've found a few tricky issues that were picked up by useful warnings, but they get buried in some of the trivial ones, so I've been trying to boil down the warnings to the ones that are potentially important